### ISSUE TITLE
When you open a saved chart/a shared link from Lightdash, it should automatically open teh chart tab

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#928]
### Context
At the moment, when you open a saved chart/a shared link from Lightdash, it opens up the `results` table by default and minimizes the other tabs. 

When I'm working on my own, I'd look at a results table when I’m not sure if my logic is correct.

E.g. I’m trying to figure out how to get the week on week change using a weird table calculation and I’m not sure about my SQL logic…in this case, I’d want to see the results

If I'm sharing a link or opening a saved chart, it's way more likely that I'm sharing a trend or want to look more into a trend I've seen. 
Basically, if I’m asking or answering a question, then I’m looking for a trend (normally), so I’d expect to see a chart, not the results table.

To put it simply:
**It’s way more likely that I want to see a chart if I'm opening a shared link/saved chart than I want to see the results table.**

### Goal
By default, if you open a saved chart or a link to an explore, we should have the chart toggle open.

**Comment by ZeRego** [comment]
@TuringLovesDeathMetal We already open the chart section when the url has a saved chart ID. 
E.g https://demo.lightdash.com/projects/3675b69e-8324-4110-bdca-059031aa8da3/saved/52416342-a6ec-45e4-92bf-451d3833527b
